src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -1179,22 +1179,7 @@ private void ParseModifiers(SyntaxListBuilder tokens, bool forAccessors)
                         break;
 
                     case DeclarationModifiers.Data:
```

Comment:
>case DeclarationModifiers.Data [](start = 20, length = 30)

Do we still need this?

Reply:
I'll be using it for the "short property syntax"